dockerfiles/ubuntu/Dockerfile

```
@@ -1,8 +1,10 @@
 FROM concourse/golang-builder as builder
 COPY . /src
 WORKDIR /src
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
```

```suggestion
WORKDIR /src
COPY go.mod .
COPY go.sum .
RUN go mod download
COPY . .
```

Same as comment above